addon/plugins/table/nodes/table.ts

```diff
@@ -146,45 +174,77 @@ export function tableNodes(options: TableNodeOptions): TableNodes {
             if (typeof node === 'string') {
               return false;
             }
-            const rdfaAttrs = getRdfaAttrs(node);
-            if (rdfaAttrs) {
-              return rdfaAttrs;
+            if (rdfaAware) {
+              return { ...getRdfaAttrs(node) };
             } else {
-              return null;
+              return { ...getClassicRdfaAttrs(node) };
             }
           },
+          contentElement: getRdfaContentElement,
         },
       ],
       toDOM(node: PNode) {
-        return [
-          'table',
-          {
-            ...node.attrs,
-            class: 'say-table',
-            style: tableStyle,
-          },
-          ['tbody', 0],
-        ];
+        if (rdfaAware) {
+          return [
+            'table',
+            {
+              ...renderRdfaAttrs(node.attrs as RdfaAttrs),
+              class: 'say-table',
+              style: `width: 100%; ${tableStyle || ''}`,
```

Comment:
I can't remember the reason why the `width: 100%;` is in the `serialize` but not the `toDOM`, but I do remember there being one... Is there a reason to change it now?

Reply:
the `toDOM` function is only used when not using the custom tableview right?

Reply:
I think so. I think the reason I didn't change this when adding `tableStyle` was that it hadn't already been set, so it's more a @dkozickis question on whether there is an edge case where this is used.